Add events:list:all command and Admin screen (#74)
AdobeDocs/commerce-events: src/pages/events/module-development.md

This topic describes how to enable your custom modules for Adobe I/O Events. You can also manually register observer events using the [`events:subscribe` command](./commands.md#subscribe-to-a-commerce-event).

## Find supported events

> Adobe Commerce is capable of emitting nearly 2,000 different observer and plugin events, but most of them aren't good candidates for use with an external App Builder application. For example, Commerce emits events before and after a customer address is loaded, saved, or deleted, but the only events of consequence are those that indicate a change of status after the address is saved or deleted.

Reviewer: There are much more than 2k different events, 2k that we think can be useful (it's already after filtering). This number can be changed as well after adding or removing some filtration.
I would say that the eventing framework supports a huge amount of different events without saying that some of them aren't good candidates.

> Adobe Commerce is capable of emitting thousands of different observer and plugin events, but most of them aren't good candidates for integrating with an external App Builder application. For example, Commerce emits events before and after a customer address is loaded, saved, or deleted, but the only events of consequence are those that indicate a change of status after the address is saved or deleted.

Reviewer 2: ```suggestion
Adobe Commerce is capable of emitting thousands of different observer and plugin events, but most of them are not good candidates to integrate with an external App Builder application. For example, Commerce emits events before and after loading, saving, or deleting a customer address, but the only consequential events are events that indicate a status change that occurs after saving or deleting the address.
```
